tools/gulp/tasks/development.ts

```diff
@@ -28,3 +37,10 @@ task(':serve:devapp', serverTask(outDir, true));
 task('serve:devapp', ['build:devapp'], sequenceTask(
   [':serve:devapp', 'material:watch', ':watch:devapp']
 ));
+
+/** Task that copies all vendors into the demo-app package. Allows hosting the app on firebase. */
+task('prepare-deploy:devapp', ['build:devapp'], () => {
```

Comment:
I use different ones since i like to have demos for different PRs going at the same time, maybe make it an argument? this looks promising: https://www.npmjs.com/package/yargs